src/client/src/containers/PostGenerationModal/index.tsx

```diff
@@ -112,57 +109,175 @@ const PostGenerationModal = ({
     if (isTemplatesFailed) {
       return formatMessage(messages.restartWizard);
     }
-    if (isTemplateGenerated) {
-      return formatMessage(messages.openInCode);
-    }
-    return (
-      <React.Fragment>
-        <Spinner className={styles.spinner} />
-        {formatMessage(messages.working)}
-      </React.Fragment>
-    );
+    return formatMessage(messages.openInCode);
   };
 
   const handleCreateAnotherProject = () => {
     resetWizard();
     history.push(ROUTES.NEW_PROJECT);
   };
 
+  const postGenMessage = () => {
+    if (isServicesSelected) {
+      return (
```

Comment:
would this look cleaner instead of the if else block?:
```
return (
      <div>
        {isServicesSelected && <p className={styles.sectionLine}>
          {formatMessage(messages.generationCompleteWithAzure)}
        </p>}
        <p className={styles.sectionLine}>
          {formatMessage(isServicesSelected ? messages.seeReadMePrefixWithAzure : messages.seeReadMePrefix)}
          <span className={styles.readMeText}>
            {formatMessage(messages.readMe)}
          </span>
          {formatMessage(messages.seeReadMeSuffix)}
        </p>
      </div>
    );
```

Reply:
Good suggestion, PR updated :)